cmd/scaffold.go

```diff
@@ -549,6 +549,7 @@ func (fnb *FlowNodeBuilder) initFvmOptions() {
 		vmOpts = append(vmOpts,
 			fvm.WithRestrictedAccountCreation(false),
 			fvm.WithRestrictedDeployment(false),
+			fvm.WithAccountStorageLimit(true),
```

Comment:
All these changes just to make this one line work :D

Reply:
Yes :). Its more to make the tests work with this one line though :).